htdocs/core/triggers/interface_50_modTicket_TicketEmail.class.php

```diff
@@ -190,6 +170,35 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
 					}
 				}
 
+				// Send email to assignee if an assignee was set at creation
+				if ($object->fk_user_assign > 0 && $object->fk_user_assign != $user->id) {
+					$userstat = new User($this->db);
+					$res = $userstat->fetch($object->fk_user_assign);
+					if ($res > 0) {
+						// Send email to notification email
+						if (empty($conf->global->TICKET_DISABLE_ALL_MAILS)) {
+							// Send email to assigned user
+							$sendto = $userstat->email;
+
+							if (!empty($conf->global->TICKET_DISABLE_MAIL_AUTOCOPY_TO)) {
+								$old_MAIN_MAIL_AUTOCOPY_TO = $conf->global->MAIN_MAIL_AUTOCOPY_TO;
+								$conf->global->MAIN_MAIL_AUTOCOPY_TO = '';
+							}
+
+							if (!empty($sendto)) {
+								$this->composeAndSendAssigneeMessage($sendto, $subject_assignee, $body_assignee, $see_ticket_assignee, $object, $langs);
+							}
+
+							if (!empty($conf->global->TICKET_DISABLE_MAIL_AUTOCOPY_TO)) {
```

Comment:
Use of $conf->global-> is forbidden.
Must use getDolGlobalString or getDolGlobalInt

Reply:
It was used a few lines ahead so I didn't think to use the function. Now fixed.